Model/lib/conifer/roles/conifer/vars/ClinEpi/production/default.yml

```diff
@@ -64,8 +64,8 @@ modelprop:
   PREFIX: "{{ prefix }}"
   VDI_CONTROL_SCHEMA: "{{ vdi_control_env_map[prefix_vdi]|default(vdi_control_env_map['default']) }}"
   VDI_DATASETS_SCHEMA: "{{ vdi_datasets_env_map[prefix_vdi]|default(vdi_datasets_env_map['default']) }}"
-  USER_DBLINK: '@{{ modelconfig_appDb_userDbLink }}'
-  ACCT_DBLINK: '@{{ modelconfig_appDb_acctDbLink }}'
+  REMOTE_STUDYACCESS_SCHEMA: '{{ remoteStudyAccessSchema }}'
```

Comment:
Why add these to the prod default.yml but not the regular one?  Either the model has these macros or it doesn't, right?